samples/sensor/bmp180/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config LOG
+	default y
+
+config LOG_PRINTK
+	default y
+
+config SENSOR_LOG_LEVEL
+	default 4
+
+# Enable I2C support by default so that the sample works with
+# the device connected either way. These defaults can be overridden if
+# needed.
+config I2C
+	default y
+
+# Enable CDC ACM UART support for debug logging for boards without
+# built-in USB support
+config ENABLE_CDC_ACM_UART
+	bool "ENABLE CDC ACM UART"
+	default y
```

Comment:
These Kconfig symbols are defined elsewhere, don't redefine them here. You can set their values for the application in a `prj.conf` file

Reply:
done.